src/XRP/raw-transaction-status.ts

```diff
@@ -39,9 +42,9 @@ export default class RawTransactionStatus {
    * Note: This constructor is exposed for testing purposes. Clients of this code should favor using a static factory method.
    */
   constructor(
-    public isValidated,
-    public transactionStatusCode,
-    public lastLedgerSequence,
-    public isFullPayment,
+    public isValidated: boolean,
+    public transactionStatusCode: string | undefined,
+    public lastLedgerSequence: number | undefined,
```

Comment:
I wasn't sure about the typing on this one. It was either `number | undefined` or `LastLedgerSequence | undefined`.

If I made it `LastLedgerSequence`, then all the tests where we pass an actual number start failing due to type incompatibility (and some places in source code where we do a numeric comparison).

Thoughts?

Reply:
It should be a number.

Reply:
TBH we might be able to remove this class. I believe it is just a bridge that was used for the legacy protocol buffers. 

Can you add a TODO for me at the class level to remove this?

Reply:
It was already there (but I changed the syntax a little so it turns highlighted yellow for me using this [TODO Highlight extension](https://marketplace.visualstudio.com/items?itemName=wayou.vscode-todo-highlight).